engine/access/subscription/tracker/base_tracker.go

```diff
@@ -155,24 +149,5 @@ func (b *BaseTrackerImpl) GetStartHeightFromLatest(ctx context.Context) (uint64,
 		return 0, err
 	}
 
-	return b.checkStartHeight(header.Height), nil
-}
-
-// checkStartHeight validates the provided start height and adjusts it if necessary.
-// If the start block is the root block, skip it and begin from the next block.
-//
-// Parameters:
-// - height: The start height to be checked.
-//
-// Returns:
-// - uint64: The adjusted start height.
-//
-// No errors are expected during normal operation.
-func (b *BaseTrackerImpl) checkStartHeight(height uint64) uint64 {
-	// if the start block is the root block, skip it and begin from the next block.
-	if height == b.rootBlockHeight {
-		height = b.rootBlockHeight + 1
-	}
-
-	return height
+	return header.Height, nil
```

Comment:
this is the main change. do not skip the root block, but instead return empty data when it is queried using `BlockExecutionData`